docker-compose.yaml

```diff
@@ -2,7 +2,7 @@ version: '3.3'
 services:
     eth-approve-bot:
         image: eth-approve-bot
-        command: python -m scripts.rewards.eth.auto_approve_eth
+        command: python -m scripts.rewards.eth.approve_eth
```

Comment:
Should we make different services for different chains? Like, can we add one more service for arb chain?

Reply:
The Docker Compose is currently only used for running the approve bot on ETH.  All other bots are run by k8s and have specific YAML files that correspond with them in `https://github.com/Badger-Finance/badger-rewards/tree/main/.deploy/base`